hazelcast/src/main/java/com/hazelcast/map/impl/operation/WANAwareOperationProvider.java

```diff
@@ -212,4 +212,10 @@ public OperationFactory createMultipleEntryOperationFactory(String name, Set<Dat
     private void checkWanReplicationQueues(String name) {
         mapServiceContext.getMapContainer(name).checkWanReplicationQueues();
     }
+
+    @Override
+    public OperationFactory createContainsValueExceptKeysOperation(String name, Object value, Set<Data> deletedKeys) {
```

Comment:
You don't need to override it since you don't do any mutations. The overrides are only for methods that mutate the map.